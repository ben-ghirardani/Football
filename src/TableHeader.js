import React, { Component } from 'react';

export default class TableHeader extends Component {

    render() {
        return(
            <tr>
                <th>Pos.</th>
                <th>Team</th>
                <th>MP</th>
                <th>W</th>
                <th>D</th>
                <th>L</th>
                <th>GF</th>
                <th>GA</th>
                <th>GD</th>
                <th>Pts</th>
            </tr>
        )
    }
 
}