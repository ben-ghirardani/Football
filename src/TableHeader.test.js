import React, { Component } from 'react';
import Enzyme, { shallow, mount, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });
import "isomorphic-fetch"

import TableHeader from './TableHeader';

describe('TableHeader', () => {

    it('should render', () => {
        const table = shallow(<TableHeader/>);
        expect(table).toMatchSnapshot();
      });

  });