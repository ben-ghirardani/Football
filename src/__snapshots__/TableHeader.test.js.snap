// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeader should render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TableHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <th>
          Pos.
        </th>,
        <th>
          Team
        </th>,
        <th>
          MP
        </th>,
        <th>
          W
        </th>,
        <th>
          D
        </th>,
        <th>
          L
        </th>,
        <th>
          GF
        </th>,
        <th>
          GA
        </th>,
        <th>
          GD
        </th>,
        <th>
          Pts
        </th>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Pos.",
        },
        "ref": null,
        "rendered": "Pos.",
        "type": "th",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Team",
        },
        "ref": null,
        "rendered": "Team",
        "type": "th",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "MP",
        },
        "ref": null,
        "rendered": "MP",
        "type": "th",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "W",
        },
        "ref": null,
        "rendered": "W",
        "type": "th",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "D",
        },
        "ref": null,
        "rendered": "D",
        "type": "th",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "L",
        },
        "ref": null,
        "rendered": "L",
        "type": "th",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "GF",
        },
        "ref": null,
        "rendered": "GF",
        "type": "th",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "GA",
        },
        "ref": null,
        "rendered": "GA",
        "type": "th",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "GD",
        },
        "ref": null,
        "rendered": "GD",
        "type": "th",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Pts",
        },
        "ref": null,
        "rendered": "Pts",
        "type": "th",
      },
    ],
    "type": "tr",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <th>
            Pos.
          </th>,
          <th>
            Team
          </th>,
          <th>
            MP
          </th>,
          <th>
            W
          </th>,
          <th>
            D
          </th>,
          <th>
            L
          </th>,
          <th>
            GF
          </th>,
          <th>
            GA
          </th>,
          <th>
            GD
          </th>,
          <th>
            Pts
          </th>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Pos.",
          },
          "ref": null,
          "rendered": "Pos.",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Team",
          },
          "ref": null,
          "rendered": "Team",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "MP",
          },
          "ref": null,
          "rendered": "MP",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "W",
          },
          "ref": null,
          "rendered": "W",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "D",
          },
          "ref": null,
          "rendered": "D",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "L",
          },
          "ref": null,
          "rendered": "L",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "GF",
          },
          "ref": null,
          "rendered": "GF",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "GA",
          },
          "ref": null,
          "rendered": "GA",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "GD",
          },
          "ref": null,
          "rendered": "GD",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Pts",
          },
          "ref": null,
          "rendered": "Pts",
          "type": "th",
        },
      ],
      "type": "tr",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
